core/src/consensus.rs

```diff
@@ -496,6 +506,9 @@ impl Tower {
             total_stake += voted_stake;
         }
 
+        let parent_is_super_oc = bank_slot == parent_slot + 1
+            && super_oc_stake as f64 / total_stake as f64 > GENESIS_VOTE_THRESHOLD;
```

Comment:
feels yucky shoving this detection in `collect_vote_lockouts`, but it does seem like the most convenient place. I can look the other way for now